### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by woodruffw** [econchick/interrogate#164]
Hi there! First of all, thank you for creating and maintaining `interrogate`! We use it on [`pip-audit`](https://github.com/pypa/pip-audit) to ensure that all of our public APIs are documented, and it's a delight to use.

On recent versions of Python (3.12+), we're seeing a `ModuleNotFoundError` due to `interrogate`'s use of `pkg_resources`, which was removed from the standard Python distribution:

```
Traceback (most recent call last):
  File "/Users/william/devel/pip-audit/env/bin/interrogate", line 5, in <module>
    from interrogate.cli import main
  File "/Users/william/devel/pip-audit/env/lib/python3.12/site-packages/interrogate/cli.py", line 11, in <module>
    from interrogate import badge_gen
  File "/Users/william/devel/pip-audit/env/lib/python3.12/site-packages/interrogate/badge_gen.py", line 11, in <module>
    import pkg_resources
ModuleNotFoundError: No module named 'pkg_resources'
```

It looks like this was patched with https://github.com/econchick/interrogate/commit/3aa94a6a9d6dca0a015d97ef389feb788cb466dd, but that that commit hasn't landed in a new release quite yet.

So: would it be possible to get a new release cut for `interrogate`?

**Comment by woodruffw** [comment]
Gentle ping on this 🙂 

(For others experiencing this: a workaround that *may* work for you is to also install `setuptools`, since `setuptools` provides `pkg_resources`.)

**Comment by s-weigand** [comment]
@econchick Any updates on this? 😅

**Comment by econchick** [comment]
finally released 1.6.0 🙈 😅 thanks for your patience!!

**Comment by woodruffw** [comment]
Thank you as well!
